Clinical-Genomics/patientMatcher
# Validate the Ensembl gene ID against the Ensembl API when converting gene symbols

### This PR adds | fixes:
- fix #213

### How to test:
- On master branch, load the demo data with the command "pmatcher add demodata"
- Notice that one of the demo patients has the gene with symbol "SKI" containing the wrong Ensembl ID
- Repeat with this branch and the gene should have the right ID


### Review:

This [version](https://semver.org/) is a:
